--- /usr/share/gems/gems/boxgrinder-build-0.10.4/lib/boxgrinder-build/helpers/guestfs-helper.rb.orig	2013-05-22 01:05:00.043819926 -0400
+++ /usr/share/gems/gems/boxgrinder-build-0.10.4/lib/boxgrinder-build/helpers/guestfs-helper.rb	2013-05-22 01:05:32.694722504 -0400
@@ -216,9 +216,8 @@
       @guestfs.aug_rm("/augeas/load//incl[. != '/etc/sysconfig/selinux']")
       @guestfs.aug_load
 
-      selinux = @guestfs.aug_get("/files/etc/sysconfig/selinux/SELINUX")
-
       begin
+        selinux = @guestfs.aug_get("/files/etc/sysconfig/selinux/SELINUX")
         @guestfs.sh("/usr/sbin/load_policy") if !selinux.nil? and !selinux.eql?('disabled')
         @log.trace "SElinux policy loaded."
       rescue

